wot-wg-2023-draft.html

```diff
@@ -87,22 +87,25 @@ <h1 id="title">PROPOSED Web of Things Working Group Charter</h1>
       the specification of building blocks
       that enable easy integration of IoT devices and services
       across IoT platforms and application domains.
-      These building blocks should complement and enhance existing standards.
-      This
-      Working Group Charter
-      covers those aspects that the
-      <a href="http://www.w3.org/groups/ig/wot">Web of Things Interest Group</a>
-      <a href="https://www.w3.org/2021/12/wot-ig-2021.html">(charter here)</a>
-      believes are mature enough to progress to W3C Recommendations.
+      These building blocks should complement and enhance existing standards and allow WoT to
+      integrate with other ecosystems and communities.
+      This reduces the initial friction for adoption and supports the further industry adoption.
+      <span class="todo">This Working Group Charter covers those aspects that the
+        <a href="http://www.w3.org/groups/ig/wot">Web of Things Interest Group</a>
+        <a href="https://www.w3.org/2021/12/wot-ig-2021.html">(charter here)</a>
+        believes are mature enough to progress to W3C Recommendations.
+      </span>
+      <!-- This is not true anymore and can be removed. IG does not write any reports. All reports, even informative ones are done by the WG.
```

Comment:
We should not preclude that the IG creates reports or initial drafts in the next charter.

Reply:
True, and it's possible.  Also, the Scripting API may follow that path some day.  But we may still want to tweak the wording here a bit as that's not the only function of the IG, and we often proceed without an IG draft.

Reply:
Scripting API is part of WG, not of IG. Please see https://www.w3.org/groups/ig/wot/publications as the IG only produces the use cases document which is never a REC in W3C. Even then, let's assume that IG decided to publish a note level document that is on a REC track. We cannot simply push it to REC in this charter if we do not state it here. So I think that there is no use to mention this and it confuses people about the relationships between the two groups.

Reply:
> Scripting API is part of WG, not of IG. Please see https://www.w3.org/groups/ig/wot/publications as the IG only produces the use cases document which is never a REC in W3C. Even then, let's assume that IG decided to publish a note level document that is on a REC track. We cannot simply push it to REC in this charter if we do not state it here. So I think that there is no use to mention this and it confuses people about the relationships between the two groups.

@egekorkan Please review the IG charter and the deliverables described there. 

> The primary deliverables of the WoT Interest Group are IG Notes that identify use cases and requirements for existing and/or new WoT building blocks (i.e., technical specifications by the WoT Working Group). The IG Notes may contain the consensus of first implementation experiences gained by the group (e.g., during PlugFests). The group will also maintain a public list of identified WoT building blocks that it is tracking and investigating.

Reply:
> @egekorkan Please review the IG charter and the deliverables described there.

@mlagally I have reviewed it again and there is no mention of a deliverable that can turn into a REC track document. I am completely fine with use cases report but it is not a deliverable that starts in IG and becomes a REC thanks to WG. It can fuel WG work with requirements though.